public/data/meet-the-talent.json

```diff
@@ -1341,6 +1341,56 @@
           "name": "LinkedIn"
         }
       ]
+    },
+    {
+      "fullName": "Jigyasu Singh Chouhan",
+      "location": "Ahmedabad, India",
+      "bio": "DevOps Engineer & Python Developer | AWS, Docker, Jenkins | CI/CD Expert | Kubernetes, Ansible, Terraform | MySQL, Python, SQL | Git, SonarQube, Grafana | AWS & Azure.",
+      "contact": {
```

Comment:
Checks failed: Email and Phone should be out side.
Please review the guidelines: 
- https://github.com/GDGAhmedabad/devfestahm?tab=readme-ov-file#list-your-profile-for-meet-the-talent